src/hardware.md

```diff
@@ -37,3 +63,271 @@ We can write import scripts to handle virtually any "open" file format that cont
+  <param name="mode_request_topic" value="robot_mode_requests"/>
+  <param name="path_request_topic" value="robot_path_requests"/>
+  <param name="destination_request_topic" value="robot_destination_requests"/>
+
+  <!-- These are the DDS specific configurations used to communicate with the clients -->
+  <param name="dds_domain" value="42"/>
+  <param name="dds_robot_state_topic" value="robot_state"/>
+  <param name="dds_mode_request_topic" value="mode_request"/>
+  <param name="dds_path_request_topic" value="path_request"/>
+  <param name="dds_destination_request_topic" value="destination_request"/>
+
+  <!-- This determines the frequency it checks for incoming state and request messages,
+  as well as how often it publishes its fleet state to RMF -->
+  <param name="update_state_frequency" value="20.0"/>
+  <param name="publish_state_frequency" value="2.0"/>
+
+  <!-- These transformations are required when the frame of the robot fleet is
+  different from that of RMF globally. In order to transform a pose from the RMF
+  frame to the free fleet robot frame, it is first scaled, rotated, then 
+  translated using these parameters -->
+  <param name="scale" value="0.928"/>
+  <param name="rotation" value="-0.013"/>
+  <param name="translation_x" value="-4.117"/>
+  <param name="translation_y" value="27.26"/>
+
+</node>
+```
+
+Furthermore, an example of this configuration can be found in the repository as well, under the packages `ff_examples_ros1` and `ff_exmaples_ros2`. This example launches the example simulation from `ROBOTIS`, shown [here](https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/#ros-1-simulation), which has a small simulated world with 3 Turtlebot3 mobile robots, each running its own ROS1 navigation stack.
+
+After successful builds for both ROS1 and ROS2 workspaces, the simulation can be launched following [these instructions](https://github.com/osrf/free_fleet#turtlebot3-simulation), which also includes a ROS2 `free_fleet` server, publishing fleet state messages and accepting mode and navigation requests over ROS2 messages and topics.
+
+#### ROS2 Navigation Stack
+
+Similar to a ROS1 navigation stack, the only difference would be the middleware used onboard the mobile robot, which in this scenario is ROS2. Unfortunately, the ROS2 `free_fleet` client is still currently under development, and this section will be updated once the refactoring, implementation and testing has been done.
+
+The same ready `free_fleet` server implementation in the repository is good to go to work in this scenario, as the interfaces provided by the fleet adapters are still the same ROS2 messages and topics.
+
+In the meantime, if required, users can also implement their own `free_fleet` client, by working with the `free_fleet` library that contains the base implementation and API for the DDS communication. This can be further elaborated in the next section [Custom Navigation Stack](#custom-navigation-stack).
```

Comment:
```suggestion
If required in the meantime, users can implement their own `free_fleet` client by working with the `free_fleet` library that contains the base implementation and API for the DDS communication. This will be further elaborated in the next section [Custom Navigation Stack](#custom-navigation-stack).
```